src/Compilers/Core/Portable/PEWriter/PeWriter.cs

```diff
@@ -115,6 +88,8 @@ private bool WritePeToStream(MetadataWriter mdWriter, Func<Stream> getPeStream,
             var managedResourceBuffer = new MemoryStream(1024);
             var managedResourceWriter = new BinaryWriter(managedResourceBuffer);
 
+            nativePdbWriterOpt?.SetMetadataEmitter(mdWriter);
```

Comment:
The callbacks are asking for the name of a type given its token, for the name of a method given a token and for the containing type token given a nested type token.

If we log the name of the method and the qualified name of its containing type in OpenMethod we should be good.